scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -126,15 +144,20 @@ def build_graph(df, dataset):
             df = df.Alias("Muon_correctedEta", "Muon_eta")
             df = df.Alias("Muon_correctedPhi", "Muon_phi")
             df = df.Alias("Muon_correctedCharge", "Muon_charge")
-        
-    #standalone quantities, currently only in data and W/Z samples
-            
+                    
     # n.b. charge = -99 is a placeholder for invalid track refit/corrections (mostly just from tracks below
     # the pt threshold of 8 GeV in the nano production)
     df = df.Define("vetoMuonsPre", "Muon_looseId && abs(Muon_dxybs) < 0.05 && Muon_correctedCharge != -99")
     df = df.Define("vetoMuons", "vetoMuonsPre && Muon_correctedPt > 10. && abs(Muon_correctedEta) < 2.4")
     df = df.Filter("Sum(vetoMuons) == 1")
-    df = df.Define("goodMuons", "vetoMuons && Muon_mediumId && Muon_isGlobal")
+    if args.trackerMuons:
```

Comment:
Is this meant to be true by default?

Reply:
No, default is global muons, we might never use tracker muons anymore but let's still keep the possibility open